ui/src/pages/frontsite/PatronProfile/PatronCurrentLoans/components/ExtendButton/ExtendButton.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
+import { Button, Icon, Popup } from 'semantic-ui-react';
+import { invenioConfig, ES_DELAY } from '@config';
+import _get from 'lodash/get';
+import _has from 'lodash/has';
+
+export const INFO_MESSAGES = {
+  hasMaxExtensions: 'You have reached the max number of extensions for a loan!',
+  hasExtendAction:
+    'Manually extending this loan, is not currently an available action!',
+  hasPendingLoans:
+    'Other users requested that book, therefore you cannot extend your loan.',
+  isOverdue: 'You loan is overdue, therefore you cannot extend the loan!',
+  isOwner: 'You are not the owner of this loan, so you cannot extend it!',
+};
```

Comment:
I would propose to change a couple of words here:
```
  hasMaxExtensions: 'You have reached the max number of extensions for <THIS> loan!',
  hasExtendAction:
    'Manually extending this loan, is not currently an available action!', -> '<It is not possible this loan>' the user does not know the technicalities of loan actions
  hasPendingLoans:
    'Other users requested that book, therefore you cannot extend your loan.', -> '<Unfortunately it is not possible to extend this loan because there are waiting requests for this literature' (to be improved probably)
  isOverdue: 'You loan is overdue, therefore you cannot extend the loan!', (do we allow this or not? maybe to be discussed)
```